### ISSUE TITLE
Ensure map slider meets colour contrast guidelines

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#940]
**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/3-a-1/

**Screen shot:**
![image](https://user-images.githubusercontent.com/43404358/94695204-8d104f00-032d-11eb-8828-ed6f18735fe6.png)

The grey (#DDD) background colour of the slider knob fails colour contrast guidelines with a ratio of 1.4:1 against the white background. The grey (#a5a5a5) border colour also fails with a ratio of 2.5:1 meaning low vision users will struggle to use this functionality.

**Solution:**
Ensure that the visual presentation of graphical components has a contrast ratio of at least 3:1 against adjacent colours so that low vision users are able to perceive the information.

**Comment by brockfanning** [comment]
This should be straightforward.
